src/screens/PostPageDetail/index.js

```diff
@@ -3,11 +3,10 @@ import {StyleSheet, View} from 'react-native';
 
 // import PostPageDetailComponent from '../../components/PostPageDetail'
 import PostPageDetailComponent from '../../components/PostPageDetail';
-import useMainPdp from './hooks/useMainPdp';
-import {CONTEXT_SOURCE} from '../../hooks/usePostContextHooks';
 import {Context} from '../../context';
 import {setFeedByIndex} from '../../context/actions/feeds';
-import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {CONTEXT_SOURCE} from '../../hooks/usePostContextHooks';
+import useMainPdp from './hooks/useMainPdp';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been rearranged and the `withInteractionsManaged` import has been removed. This is a significant change as it affects how interactions are managed in the `FeedsPostDetail` component. Please ensure that this does not negatively impact the user experience, especially if your application relies on managing interactions for performance reasons.
```diff
- import useMainPdp from './hooks/useMainPdp';
- import {CONTEXT_SOURCE} from '../../hooks/usePostContextHooks';
- import {Context} from '../../context';
- import {setFeedByIndex} from '../../context/actions/feeds';
- import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+ import PostPageDetailComponent from '../../components/PostPageDetail';
+ import {Context} from '../../context';
+ import {setFeedByIndex} from '../../context/actions/feeds';
+ import {CONTEXT_SOURCE} from '../../hooks/usePostContextHooks';
+ import useMainPdp from './hooks/useMainPdp';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->